ui/job-view/headerbars/WatchedRepo.jsx

```diff
@@ -127,23 +125,19 @@ class WatchedRepo extends React.Component {
   };
 
   render() {
-    const { repoName, unwatchRepo, repo, pushRoute } = this.props;
+    const { repoName, unwatchRepo, repo } = this.props;
     const { status, messageOfTheDay, reason, statusInfo } = this.state;
     const watchedRepo = repo.name;
     const activeClass = watchedRepo === repoName ? 'active' : '';
     const { btnClass, icon, color } = statusInfo;
     const pulseIcon = statusInfo.pulseIcon || null;
 
     return (
-      <ButtonGroup>
-        <Button
-          onClick={() => {
-            pushRoute({ search: updateRepoParams(watchedRepo) });
-            this.updateTreeStatus(true);
-          }}
-          className={`btn-view-nav ${btnClass} ${activeClass}`}
+      <div className="btn-group" role="group">
```

Comment:
`ButtonGroup` is replaced by raw `div` for the following reasons:
  * to have single root element for this (there are `Link` and `UncontrolledDropdown` inside it, and we need element that wraps them)
  * to keep the same styling as the current code